content/recipes/introduction/priorization.md

```diff
@@ -0,0 +1,207 @@
+
+When mentioned data accessibility, this could be broadly classified into three categories:
+- **Accessing via web-interface** - Here, the data is hosted by the data owner in a cloud system and the relevant data can be accessed using the web simply.
+- **Programmatic access** - Here, the data is made available via programmatic services such as REST API. This is mainly of interest to data handlers within the FAIR expert team who can with few lines of programmatic code access the data.
+- **No access** - Here, given the confidentiality and legal aspects involved in dealing with the data, certain data owners will be reluctant to provide access to the data. One such example in this case could be a potential lead compound in a pharmaceutical company. Here, the company would give no access to the data until they have patented the compound or given the compound to clinical trial.
+
+**In summary, projects that FAIR experts can get access to data should be at a higher priority than those that have limitations on data access.**
+
+#### 6. Prioritisation based on presence of data champions
+
+**Data champions** can be defined as a group of people that have an expertise in dealing with certain processes, data, or pipeline. When dealing with specific processes or data types, it is beneficial if an expert who has experience in dealing with such processes or data already exists within the FAIR expert team.
+
+**Prioritisation of projects with consideration of expertise required in dealing with FAIRification tasks should be done.** Such projects can then have a predetermined timeline involved, making the step-by-step approach for accomplishing a FAIR process or data comparatively easy.
+
+#### 7. Prioritisation based on data types
+[comment]: <TODO!!>
+Each data can be available in a number of different formats like sequencing data can be in INSDC SRA format (some cookbook recipe?), or imaging data in DICOM format and more examples here. 
+- Community approved data type format (FAIRSharing)
+- Is there any repository accepting this data type (Array Express)
+
+
+
+Along with the above criteria, two more factors play a crucial role in the prioritisation process. The first being the **time management**. Since the FAIRification process can depend on a number of factors, it is important to acknowledge the time required for the process to complete. The second being **risk management**, with the assessment of potential roadblocks that could interfere essential. Such roadblocks should be identified and rectified if possible prior to starting a FAIRification process. 
```

Comment:
```suggestion
Along with the above criteria, two more factors play a crucial role in the prioritisation process:

- [ ]  The first is the **time management**. Since the FAIRification process can depend on a number of factors, it is important to acknowledge the time required for the process to complete. 

- [ ] The second is  **risk management**, with the assessment of potential roadblocks that could interfere essential. Such roadblocks should be identified and rectified if possible prior to starting a FAIRification process. 

```